_docs/index.md

```diff
@@ -12,6 +12,8 @@ type: markdown
 
 # Welcome
 
+Here you will find the latest ({{site.data.istio.version}}) documentation, follow this link for the [documentation for the latest stable release (0.1)](https://istio.io/v-0.1/docs/)
```

Comment:
good idea. see new rev - how about now ? also any idea how/where to make a real nav bar - or is this enough?